src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```
@@ -1023,6 +1127,12 @@ public int getMaxHeight()
 		return maxHeight;
 	}
 
+	@Override
+	public @NotNull BiomeProvider vanillaBiomeProvider()
+	{
+		return null;
```

Should throw an exception